### ISSUE TITLE
Transparent Spinner without label.

### ISSUE THREAD

**Issue by esafwan** [Paldom/SpinnerDialog#41]
I am trying to show the spinner without any background color or text. Like in this image

![Transparent Spinner](http://plugins.telerik.com/images/default-source/plugins/spinner/android-screenshot-1.png?sfvrsn=5)

Is it possible with this plugin?

**Comment by vitormalencar** [comment]
Any Update for this ?
 i have the same issue

**Comment by AmitMY** [comment]
Any update for this?
I have the same issue

**Comment by greybax** [comment]
yes, its possible. I've fixed it and forked with merged all PR in branch here [cordova-plugin-spinnerdialog](https://github.com/greybax/cordova-plugin-spinnerdialog)

**Comment by AmitMY** [comment]
Can u make a PR for this repo?

**Comment by greybax** [comment]
Does it make sense? Seems that this report is dead. You can fork [my repo](https://github.com/greybax/cordova-plugin-native-spinner) or any another from here https://github.com/Paldom/SpinnerDialog/network

**Comment by AmitMY** [comment]
@greybax Only reason I have is that this repo is Ionic supported, meaning it is published on their site, and has a nice wrapper for injecting to angular.
@Paldom Will you be available to merge such PR? / give @greybax some control over the repo so he can merge and fix it?
